pkg/integrations/v4/dm/emitter.go

```diff
@@ -53,6 +55,8 @@ type emitter struct {
 	reqsQueue            chan fwrequest.FwRequest
 	reqsToRegisterQueue  chan fwrequest.EntityFwRequest
 	reqsRegisteredQueue  chan fwrequest.EntityFwRequest
+	retryBo              *backoff.Backoff
+	maxRetryBo           int
```

Comment:
pls specify unit:
- either as suffix `retryBOSecs`
- or change type to time.Duration